distribution/docs/src/main/jdocs/content/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -0,0 +1,333 @@
+
+. On the *Results* tab, select a search fr4om the drop-down list.
+. Perform any of these action upon the results list of the selected query:
+.. Filter the result set locally (does not re-execute the search),
+.. Customize results sorting (Default: Title in Ascending Order).
+.. Toggle results view between *List* and *Gallery*.
+
+==== Bookmarks Tab
+
+Search Results that have been bookmarked can be filtered and sorted.
+They are denoted with a *star* icon. (image:bookmarked-icon.png[]
+)
+
+.Bookmarked Result Options
+* *Add to Bookmarks*: Adds the result to bookmarks.
+** Changes to *Remove from Bookmarks* for bookmarked items: Removes the result from bookmarks.
+* *Hide from Future Searches*: Adds to a list of results that will be hidden from future searches.
+* *Expand Metacard View*: Navigates to a view that only focuses on this particular result.
+* *Download*: Downloads the result's associated product directly to local machine.
+
+=== Viewing Search Results
+
+On the *Result Preview* pane, view a summary of information about the selected resource.
+The *Details* tab provides a list of specific metadata.
+
+<<_viewing_revision_history,History>>:: View revision history of this resource.
+<<_editing_associations_on_a_record,Associations>>:: View or edit the relationship(s) between this resource and others in the catalog.
+<<_viewing_metadata_quality,Quality>>:: View the completeness and accuracy of the metadata for this resource.
+<<_exporting_a_record,Actions>>:: Export the metadata/resource to a specific format.
+<<_archiving_a_record,Archive>>:: Remove the selected record from standard search results.
+<<_overwriting_a_record,Overwrite>>:: Overwrite a record.
+
+==== Editing Records
+
+Records can be edited from the *Summary* or *Details* tabs.
+
+==== Editing Associations on a Record
+
+Update relationships between records through *Associations*.
+
+. Select the desired record from the *Results* tab.
+. Select the *Associations* tab.
+. Select *Edit*.
+. For a new association, select *Add Association*. (Only items in the current result set can be added as associations).
+.. Select the related record from either the *Parent* or *Child* menu.
+.. Select the type of relationship from the *Relationship* menu.
+.. Select *Save*.
+. To edit an existing association, update the selections from the appropriate menu and select *Save*.
+
+View a graphical representation of the associations by clicking the *graph* icon(image:graph-icon.png[graph icon]
+).
+
+==== Viewing Revision History
+
+View the complete revision history of a record.
+
+. Select the desired record from the *Results* tab.
+. Select the *History* tab.
+.. Select a previous version from the list to preview.
+.. If desired, select *Revert to Selected Version* to undo changes made after that revision.
+
+==== Viewing Metadata Quality
+
+View and fix issues with metadata quality in a record.
+
+[NOTE]
+====
+Correcting metadata issues may require heightened permissions.
```

Comment:
Of those, I like `elevated` best. Maybe `administrative privileges` is more familiar to the audience?